sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ClientFactory.cs

```diff
@@ -189,12 +189,12 @@ internal static TokenCredential CreateCredential(IConfiguration configuration)
                     string.IsNullOrWhiteSpace(clientId) ||
                     string.IsNullOrWhiteSpace(azureCloud))
                 {
-                    throw new ArgumentException("For managed identity as a federated identity credential, 'tenantId', 'clientId', 'azureCloud', and one of ['managedIdentityClientId', 'resourceId', 'objectId'] must be specified via environment variables or the configuration.");
+                    throw new ArgumentException("For managed identity as a federated identity credential, 'tenantId', 'clientId', 'azureCloud', and one of ['managedIdentityClientId', 'managedIdentityResourceId', 'managedIdentityObjectId'] must be specified via environment variables or the configuration.");
```

Comment:
This isn't correct.   The config names are "resourceId" and "objectId".    We did not add overloads for them with "managedIdentity" as a prefix and we cannot rename the existing ones.